reportportal_client/service.py

```diff
@@ -189,7 +189,13 @@ def start_test_item(self, name, start_time, item_type, description=None,
             }
         """
         if parameters is not None:
-            parameters = [{"key": key, "value": str(value)}
+            def _str(value):
+                if isinstance(value, unicode):
```

Comment:
You will get the following error with Python 3:
NameError: name 'unicode' is not defined

Reply:
Sure, thank you!